### ISSUE TITLE
Window Resizing Issue

### ISSUE THREAD

**Issue by AdrianZZBeng** [iina/iina#3252]
- macOS 10.13.5
- IINA 1.1.2 Build 128

Attempting to resize/enlarge any window beyond the limits of my viewport/screen (as I often do in VLC) results in the window immediately resizing/shrinking and repositioning along the bottom of the screen. I have disabled window resizing in the UI settings but this still happens, and cannot find any other obvious setting which would automatically reset the window's dimensions like this.

Is this a bug or am I doing something dumb? Any ideas?

Many thanks!

**Comment by anohren** [comment]
Do you mean that you end up with a window whose edges you moved are constrained to the bounds of the screen in which the window resides? Or does the entire window move and reset to its initial size?

**Comment by AdrianZZBeng** [comment]
> Do you mean that you end up with a window whose edges you moved are constrained to the bounds of the screen in which the window resides? Or does the entire window move and reset to its initial size?

Yes to the first one - the window's edges become constrained by the bounds of the screen no matter how large I attempt to make the window

**Comment by ltecheroffical** [comment]
I am having the same issue, with the window shrinking, that kinda makes me want to go back to VLC because I like small windows the way I want and I don't want it changing every time I play a video!
